src/common/config_parser.cpp

```diff
@@ -255,8 +255,13 @@ void ConfigParser::addOptionsTraining(cli::CLIWrapper& cli) {
       "If these files do not exist they are created");
 #ifdef USE_SENTENCEPIECE
   cli.add<std::vector<float>>("--sentencepiece-alphas",
-                              "Sampling factors for SentencePieceVocab;"
-                              "i-th factor corresponds to i-th vocabulary");
+      "Sampling factors for SentencePieceVocab; i-th factor corresponds to i-th vocabulary");
```

Comment:
SentencePieceVocab is from our code, not a term used in sentence piece toolkit, so I would prefer to use something less confusing for users, for instance, "SentencePiece vocabulary".